stable/listmonk/values.yaml

```diff
@@ -0,0 +1,84 @@
+replicaCount: 2
+
+image:
+  repository: listmonk/listmonk
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  # tag: ""
```

Comment:
Can we include a tag?

Reply:
I thought by default it would be better to get image version from Chart.yaml, wdyt?

Reply:
Oh yes, that's better!